src/meetings/components/meetingActionsBar/RaiseHandButton.tsx

```diff
@@ -38,12 +38,12 @@ const RaiseHandButton = (): ReactElement | null => {
 	const createSnackbar: CreateSnackbarFn = useSnackbar();
 
 	const toggleRaiseHand = useCallback(() => {
-		MeetingsApi.raiseHand(meetingId!, !iHaveHandRaised);
-	}, [iHaveHandRaised, meetingId]);
+		MeetingsApi.raiseHand(meetingId!, !iHaveHandRaised, sessionId);
```

Comment:
When I lower my hand, this snackbar appears:
<img width="378" alt="Screenshot 2025-04-15 alle 10 45 41" src="https://github.com/user-attachments/assets/4f9d7ac4-35b5-436f-94b3-fc756650ca68" />

To avoid it, don't pass `sessionId`